pkgs/development/mobile/gomobile/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, lib, fetchgit, buildGoModule, zlib, makeWrapper, xcodeenv, androidenv }:
+
+{ xcodeWrapperArgs ? { }
+, xcodeWrapper ? xcodeenv.composeXcodeWrapper xcodeWrapperArgs
+, androidPkgs ? androidenv.androidPkgs_9_0.androidsdk }:
+
+buildGoModule {
+  pname = "gomobile";
+  version = "2020-06-22";
+
+  vendorSha256 = "1n1338vqkc1n8cy94501n7jn3qbr28q9d9zxnq2b4rxsqjfc9l94";
+
+  src = fetchgit {
+    # WARNING: Next commit removes support for ARM 32 bit builds for iOS
+    rev = "33b80540585f2b31e503da24d6b2a02de3c53ff5";
+    name = "gomobile";
+    url = "https://go.googlesource.com/mobile";
+    sha256 = "0c9map2vrv34wmaycsv71k4day3b0z5p16yzxmlp8amvqb38zwlm";
+  };
+
+  subPackages = [ "bind" "cmd/gobind" "cmd/gomobile" ];
+
+  # go: cannot find GOROOT directory: go
+  doCheck = false;
+
+  patches = [ ./resolve-nix-android-sdk.patch ];
+
+  buildInputs = [ makeWrapper ] ++ lib.optional stdenv.isDarwin xcodeWrapper;
```

Comment:
That should be most likely nativeBuildInputs.

Reply:
You're right. Changed.